inst/tests/tests.Rraw

```diff
@@ -3737,23 +3737,6 @@ test(1144.4, DT[, list(lx=x[.N]), by=x], data.table(x=DT$x, lx=DT$x))
 ans = copy(DT)
 test(1144.3, DT[,`:=`(lx=tail(x,1L)), by=y], ans[, lx := x])
 
-# FR #2356 - retain names of named vector as column with keep.rownames=TRUE
-x <- 1:5
-setattr(x, 'names', letters[1:5])
-test(1144.1, as.data.table(x, keep=TRUE), data.table(rn=names(x), x=unname(x)))
```

Comment:
@mattdowle e.g. this one, see directly above is `test(1144` for a completely different issue and directly below is `test(1145`.

Bumping 1145 to the end would have less of a diff I guess?

Reply:
Nice find. Seems like the original merge was half-right in that there's a gap for 1143 between 1142 and the 1st 1144. So it'd be better to bump the first 1144.* back to 1143 and then leave the 2nd 1144.* where it is.

Reply:
Oh, good catch, I didn't plumb around for gaps too much. Will do & squash.